emcc.py

```diff
@@ -1423,9 +1423,6 @@ def check(input_file):
       if options.separate_asm:
         exit_with_error('cannot --separate-asm when emitting wasm, since not emitting asm.js')
 
-      if '-fsanitize-minimal-runtime' in newargs:
-        shared.Settings.UBSAN_RUNTIME = 1
```

Comment:
Can we completely remove this setting then?

Reply:
Yes, that was the intention. I thought I did it already. It will be done in the next update.